examples/bert/bert_ptq_cpu.json

```diff
@@ -59,6 +60,7 @@
                 "quant_preprocess": true,
                 "per_channel": false,
                 "reduce_range": false,
+                "calibrate_method": "MinMax",
```

Comment:
add TODO here

Reply:
hard to comment for Json, I add some comment here.
![image](https://github.com/microsoft/Olive/assets/13343117/2865cba9-b032-400c-b38a-f9ba042339e3)